src/Microsoft.OData.Core/UriParser/Visitors/ISyntacticTreeVisitor.cs

```diff
@@ -145,19 +145,26 @@ public interface ISyntacticTreeVisitor<T>
         T Visit(FunctionParameterToken tokenIn);
 
         /// <summary>
-        /// Visits a AggregateToken
+        /// Visits a AggregateTransformationToken
         /// </summary>
-        /// <param name="tokenIn">The AggregateToken to bind</param>
-        /// <returns>A T node bound to this AggregateToken</returns>
+        /// <param name="tokenIn">The AggregateTransformationToken to bind</param>

```

Comment:
>AggregateTransformationToken [](start = 38, length = 28)

<param> tag doesn't match parameter type; this function actually does visit an AggregateToken.